css/css-ui/outline-018.html

```diff
@@ -16,28 +16,43 @@
   }
 }
 
-#test {
-  animation: outline-anim 3s -1.5s paused linear;
+#test1 {
+  animation: outline-anim 3s 0s paused linear;
+  outline: solid 1px black;
+}
+#test2 {
+  animation: outline-anim 3s -1s paused linear;
+  outline: solid 1px black;
+}
+#test3 {
+  animation: outline-anim 3s -2s paused linear;
+  outline: solid 1px black;
+}
+#test4 {
+  animation: outline-anim 3s 0s paused reverse-linear;
   outline: solid 1px black;
 }
 </style>
 <body>
-  <div id="test"></div>
+  <div id="test1"></div>
+  <div id="test2"></div>
+  <div id="test3"></div>
+  <div id="test4"></div>
   <div id=log></div>
 
   <script>
-    test(
-      function(){
-        var test = document.getElementById("test");
-
-        test.style.animation = "outline-anim 3s 0s paused linear"
-        assert_equals(getComputedStyle(test).outlineStyle, 'solid');
-        test.style.animation = "outline-anim 3s -1s paused linear"
-        assert_equals(getComputedStyle(test).outlineStyle, 'solid');
-        test.style.animation = "outline-anim 3s -2s paused linear"
-        assert_equals(getComputedStyle(test).outlineStyle, 'dotted');
-        test.style.animation = "outline-anim 3s 0s paused reverse-linear"
-        assert_equals(getComputedStyle(test).outlineStyle, 'dotted');
-      }, "outline-style is animated as a discrete type");
+    setup({ explicit_done: true });
```

Comment:
Likewise: it would seem better to use `async_test` for this rather than `test` with `explicit_done`.

Reply:
I just reconsidered it, but after starting to rewrite this way, I realized this doesn't show the intent really well. These tests are synchronous, they are just delayed to allow the animation-sync to happen. If you use async_test and use a requestAnimationFrame in each of the test, each test is going to wait and it's fuzzier why this happens.

I'd be happy to add a comment explaining this in the file though if you believe that is a good idea.